app/helpers/service_helper.rb

```diff
@@ -6,7 +6,7 @@ def default_title
   def goods_nomenclature_title(goods_nomenclature)
     t(
       'title.goods_nomenclature',
-      goods_description: goods_nomenclature.description_plain,
+      goods_description: goods_nomenclature.formatted_description,
```

Comment:
So I think the history of this is:

1. Ata added a new attribute to the backend which removed certain characters from the description that shows in the title (he chose to call this `description_plain`)
2. He updated the frontend to reflect this for all titles for all goods nomenclature
3. For some reason its not working specifically for headings
4. Sections seem to be unaffected as far as I can see on staging

So I think we still need `description_plain` everywhere but could do with knowing why it isn't showing properly for headings

Looking at the v2 api response from the backend (which is what the frontend consumes) I can see the `description_plain` attribute is missing https://dev.trade-tariff.service.gov.uk/api/v2/headings/0101

So my best guess is that simply updating the backend application to properly surface the `description_plain` attribute should resolve this issue on the frontend

The method is implemented on the parent class of all goods nomenclature (Chapter, Heading, Subheading, Commodity are all instances of this class) https://github.com/trade-tariff/trade-tariff-backend/blob/main/app/models/goods_nomenclature.rb#L30-L31C45 and surfaced to the api resource via https://github.com/trade-tariff/trade-tariff-backend/blob/main/app/serializers/api/v2/headings/heading_serializer.rb#L11-L16

So we should safely be able to add `description_plain` to the list of attributes :)

Reply:
Thanks Will, i can see that its missing like you described below but it would still mean that i have to change how the description_plain is being presented as currently its all in capitals right?

Reply:
I think the worry with changing the API was more that if something is consuming it and expects capitals then it could cause issues

Reply:
Good shout re the api users expecting the capitalised form. Technically the description_plain field actually predates Ata's change and I believe he merely extended it.

We will want to make sure that the Heading, Subheading and Commodity goods nomenclatures do use the description plain since the alternative is having weird characters in the titles and at least 11468 goods nomenclatures on the XI service will be affected if we don't